__tests__/Unit/Components/Tasks/TaskDetails.test.tsx

```diff
@@ -623,11 +622,12 @@ describe('Task Details > Task Request', () => {
         fireEvent.click(taskRequestButton);
 
         await waitFor(() => {
-            screen.getByText(/successfully requested for task/i);
+            const taskRequestModalTitle = screen.getByText(/Task Request/i);
+            expect(taskRequestModalTitle).toBeInTheDocument();
         });
     });
 
-    it('Error toast should be shown on error', async () => {
+    test('Error toast should be shown on error', async () => {
```

Comment:
(Not a change request) when should we use test and when it?

Reply:
I think they both behave the same. Anyways changed it back to it